docs/solutions/sql_server_always_on_availability_groups/include.tf

```diff
@@ -0,0 +1,55 @@
+// Values below are retrieved from terraform.tfvars file
+// You probably don't need to change anything here.
+
+variable "tenancy_id" {}
+
+variable "compartment_id" {}
+variable "user_id" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "private_key_password" {}
+variable "region" {}
+
+variable "home_region" {}
+
+//True by default. Solve inconsistency problems but creates performance issue on destroy.
```

Comment:
The perf issue on destroy has been fixed for a while - let's get rid of this variable.